src/Components/DetailsPage/DetailsReccInterface.js

```diff
@@ -6,48 +6,53 @@ import "./_DetailsReccInterface.scss"
 import DetailsFriendAvatar from "./DetailsFriendAvatar"
 import paperPlane from "../../images/paper-plane.png"
 
-const DetailsReccInterface = ({ id }) => {
-  const {
-    currentUser
-  } = useContext(UserContext)
-  
-  const  { changeModalState, changeModalShow } = useContext(RecModalContext)
-
-  const friendAvatars = currentUser.recommendations
-    .reduce((acc, recc) => {
-      const isDuplicateRecc = acc.some(accRecc => accRecc.recommender.id === recc.recommender.id)
-      if (recc.show.tmdbId === id && !isDuplicateRecc) acc.push(recc)
-      return acc
-    }, [])
+const DetailsReccInterface = ({ id, recommenders }) => {
+  const { changeModalState, changeModalShow } = useContext(RecModalContext)
+
+  const addlRecommendersMsg = <p className="addl-recommenders">and other friends</p>
```

Comment:
This is a great idea for when there are lots of recommenders! It keeps the details page from being too crowded.